projects.js

```diff
@@ -0,0 +1,148 @@
+    languages: ['Ruby on rails', 'CSS', 'JavaScript', 'HTML'],
+    seeProject: 'See Project',
+    Links: ['codekit', 'Github', 'Javascript', 'Bootstrap', 'Terminal', 'Codepen'],
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry",
+    title: 'Keeping track of hundreds of components',
+    liveLink: 'https://github.com/Prisca-tech/Portfolio',
+    sourceLink: 'https://github.com/Prisca-tech',
+  },
+
+  //  card six
+  {
+    image: './Image/Container/Snapshoot Portfolio.svg',
+    articleTitle: 'Multi-Post Stories',
+    languages: ['Ruby on rails', 'CSS', 'JavaScript', 'HTML'],
+    seeProject: 'See Project',
+    Links: ['codekit', 'Github', 'Javascript', 'Bootstrap', 'Terminal', 'Codepen'],
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry",
+    title: 'Keeping track of hundreds of components',
+    liveLink: 'https://github.com/Prisca-tech/Portfolio',
+    sourceLink: 'https://github.com/Prisca-tech',
+  },
+];
+
+const projectContainer = document.getElementById('project-container');
+
+const projectCards = project.map((card) => ` <div class="projects">
+      <div class="project-image"></div>
+      <div class="project-description">
+        <h4>
+          ${card.articleTitle} <br />
+          Gain+Glory
+        </h4>
+        <ul class="languages">
+          ${card.languages.map((technology) => `<li>${technology}</li>`).join('')}
+        </ul>
+        <button type="button" class="button modal-button" id="modal-btn">See Project</button>
+        </div>
+    </div>`).join('');
+
+projectContainer.innerHTML += projectCards;
+
+const cardModal = project.map((card) => ` <div class="project-card-bg">
+      <div class="project-card">
+        <button type="button" class="close-project"><i class="fa-solid fa-xmark"></i></button>
+        <div class="project-thumbnail">
+          <img src="${card.image}" alt="">
+        </div>
+        <div class="project-info">
+          <div class="heading-tools">
+            <h4 class="project-card-heading">${card.title}</h4>
+            <ul class="flex-list">
+             ${card.Links.map((technology) => `<li>${technology}</li>`).join('')}
+            </ul>
+          </div>
+        </div>
+        <p class="project-description">${card.description}</p>
+        <div class="project-buttons">
+            <a  class="button project-button" href="${card.liveLink}">See Live
+              <img src="./Image/live-icon.svg" alt="">
+            </a>
```

Comment:
Please add the proper icon for the `See Live` button to match the Figma design.

| Your Design | Figma Design |
|-------------|---------------|
| <img width="274" alt="image" src="https://user-images.githubusercontent.com/34578631/175136444-fd744d56-7c23-45b5-bdeb-b2a885da3d11.png"> | <img width="274" alt="image" src="https://user-images.githubusercontent.com/34578631/175136473-eca8a94c-a23c-47ce-8010-d764b78703c6.png"> |